src/Microsoft.PowerShell.Commands.Management/commands/management/GetChildrenCommand.cs

```diff
@@ -172,6 +172,25 @@ public uint Depth
         }
 
         /// <summary>
+        /// Gets or sets whether symbolic links are to be followed when recursing.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter FollowSymlink
+        {
+            get
+            {
+                return _followSymlink;
+            }
+            set
+            {
+                _followSymlink = value;
+                if (value == true)
+                    Recurse = true;
```

Comment:
It looks like that was in response to a specific bug. In our case it's just part of a new feature.